lib/fetch/index.js

```diff
@@ -122,130 +122,178 @@ class Fetch extends EE {
   }
 }
 
+const channels = {}
+
+try {
+  const diagnosticsChannel = require('diagnostics_channel')
+  channels.start = diagnosticsChannel.channel('undici:fetch:start')
+  channels.end = diagnosticsChannel.channel('undici:fetch:end')
+  channels.asyncStart = diagnosticsChannel.channel('undici:fetch:asyncStart')
+  channels.asyncEnd = diagnosticsChannel.channel('undici:fetch:asyncEnd')
+  channels.error = diagnosticsChannel.channel('undici:fetch:error')
+} catch {
+  channels.start = { hasSubscribers: false }
+  channels.end = { hasSubscribers: false }
+  channels.asyncStart = { hasSubscribers: false }
+  channels.asyncEnd = { hasSubscribers: false }
+  channels.error = { hasSubscribers: false }
+}
```

Comment:
Can you add them to https://github.com/nodejs/undici/blob/main/lib/core/diagnostics.js?

Reply:
added the channels to diagnostics.js